### PR TITLE
Bump adal4j version to 1.6.7

### PR DESCRIPTION
**Pull request by sberyozkin**

### CODE REVIEW

File: extensions/jdbc/jdbc-mssql/runtime/pom.xml [hunk 1]
```diff
@@ -29,50 +29,10 @@
         </dependency>
         <!--
             adal4j isn't used directly but you need to have it around for compilation to succeed.
-            Since that's the only reason, doesn't seem necessary to require the specific version in the shared bom.
-            -->
+        -->
```
Comment (Sanne) [inline extensions/jdbc/jdbc-mssql/runtime/pom.xml:32]: 👍

**Review by Sanne — APPROVED** [review]

**Comment by sberyozkin** [comment]
Native/Misc4 tests failed, but it does not seem to be related, a repository download failure

**Comment by Sanne** [comment]
ouch - I pressed the "update branch" button thinking it would rebase and therefore trigger a new build but it seems it has actually merged `main` into this branch. [Merge branch 'main' into mssql-adal4j-1.6.7](https://github.com/quarkusio/quarkus/pull/24079/commits/3cfe23e9841be29c5b4e840c5d56357468f14b9e)

Is that OK and expected @gsmet ?

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
So first, as explained in my email, let's not abuse the update branch button.
Second, as explained in my email ;), you need to specifically choose Update with rebase. It's not the default and you cannot change the default.

I will rebase.

**Comment by Sanne** [comment]
@gsmet thanks and sorry :)

**Comment by sberyozkin** [comment]
JDK17 JVM test failure:
```
Running 2/2. 1 failure so far. Running: io.quarkus.vertx.http.testrunner.SimpleET#testGreetingEndpoint()
2022-03-04 09:50:51,037 INFO  [io.quarkus] (Test runner thread) Quarkus(test application) stopped in 0.029s
2022-03-04 09:50:51,054 ERROR [io.qua.test] (Test runner thread) ==================== TEST REPORT #1 ====================
2022-03-04 09:50:51,054 ERROR [io.qua.test] (Test runner thread) Test SimpleET#testHelloEndpoint() failed 
: java.lang.AssertionError: 1 expectation failed.
Expected status code <200> but was <404>.
```
Not related (looks like the race condition, the endpoint is not up yet)

**Pull request merged**